tests/ndimage/footprint_filter_test.py

```diff
@@ -0,0 +1,192 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+import numpy as np
+import scipp as sc
+import scipy.ndimage
+from scipp.ndimage import (generic_filter, maximum_filter, median_filter,
+                           minimum_filter, percentile_filter, rank_filter)
+import pytest
+
+filters = (generic_filter, maximum_filter, median_filter, minimum_filter,
+           percentile_filter, rank_filter)
+
+
+def make_histogram2d():
+    da = sc.DataArray(sc.array(dims=['x', 'y'], values=np.random.rand(100, 100)))
```

Comment:
Not seeding the rng might make the tests flaky and unreproducible. Unless you print `da` in its entirety.

Reply:
Does that apply in this case? We are not checking concrete values, only compare to scipy.

Reply:
Does it affect conversion of variables in arguments to integers? I don't think so? So you are right, it shouldn't matter.